src/hooks/useWalletBookmarkedProjects.ts

```diff
@@ -0,0 +1,53 @@
+import { useSupabaseClient } from '@supabase/auth-helpers-react'
+import { useEffect, useState } from 'react'
+import { Database } from 'types/database.types'
+
+import { useDBProjectsQuery } from './useProjects'
+
+export function useWalletBookmarkedProjects({
+  wallet,
+}: {
+  wallet: string | undefined
+}) {
+  const [createdAtIds, setCreatedAtIds] = useState<{ [id: string]: number }>({})
+
+  const supabase = useSupabaseClient<Database>()
+
+  useEffect(() => {
+    supabase
```

Comment:
how does this actually work under the hood? How are we able to execute an sql query on the frontend?

Reply:
should we not be able to? i've only used it locally so maybe it would break in prod. can always create an api endpoint for it

Reply:
im honestly just curious how this actually works haha. Like what's actually happening here? Does supabaseClient send this query to the server via http?

Reply:
iiuc that's the job of supabaseClient. we configure it with supabase .env vars so it can make those requests

Reply:
ya haha all good. i was just curious how it actulaly works. and it works how i expected: they build a query string out of the 'sql query', fire off a HTTP req to the supabase server and it gives back the db query results. 

<img width="611" alt="Screenshot 2023-06-12 at 5 01 16 PM" src="https://github.com/jbx-protocol/juice-interface/assets/12551741/e674b2b9-232e-43f8-a15c-483dcbf9e8dc">